clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/HealthState.groovy

```diff
@@ -2,5 +2,5 @@
 package com.netflix.spinnaker.clouddriver.model
 
 enum HealthState {
-  Up, Down, Unknown, Starting, OutOfService
+  Up, Down, Unknown, Starting, OutOfService, Succeeded, Failed
```

Comment:
I added these two states in anticipation of short running jobs, which can end successfully or fail. @ajordens, @cfieber, @tomaslin WDYT?